pkg/cloud/aws/assets/deployment.yaml

```diff
@@ -55,6 +55,9 @@ spec:
         - mountPath: /etc/kubernetes
           name: host-etc-kube
           readOnly: true
+        - name: trusted-ca
+          mountPath: /etc/pki/ca-trust/extracted/pem
+          readOnly: true
```

Comment:
Does this need to be marked optional?

Reply:
I don't think so, since system CA will be written to this config-map anyway.